schemas/deviceToCloud/cell_position/cell-position.json

```diff
@@ -11,12 +11,32 @@
             "type": "string",
             "const": "DATA"
         },
-        "data": {
+        "config": {
             "type": "object",
             "properties": {
                 "doReply": {
                     "$ref": "#/definitions/DoReply"
                 },
+                "highConfidence": {
```

Comment:
"hiC" or something shorter to save some bytes?

Reply:
hiConf?